carto/import_api.py

```diff
@@ -113,6 +116,25 @@ def __init__(self, url, client, api_version='v1', interval=None, **kwargs):
         super(URLImport, self).__init__(client, **kwargs)
 
 
+class CartoExportJob(ImportJob):
```

Comment:
Why not calling this class simply ExportJob? None of the other classes have "Carto" in their names.